test/test_load_constructors.jl

```diff
@@ -4,7 +4,7 @@
     op_model = OperationModel(TestOptModel, PM.DCPlosslessForm, c_sys5)
     @test_logs (:warn, warn_message) construct_device!(op_model, :Load, model);
     model = DeviceModel(PSY.PowerLoad, PSI.DispatchablePowerLoad)
-    warn_message = "The Formulation DispatchablePowerLoad only applies to Controllable Loads, \n Consider Changing the Device Formulation to StaticPowerLoad"
+    warn_message = "The Formulation DispatchablePowerLoad only applies to FormulationControllable Loads, \n Consider Changing the Device Formulation to StaticPowerLoad"
```

Comment:
IMHO this is unnecessarily testing an implementation detail. If some ever changes the warning message, like say removing the oddly placed "\n" then they will have to also change the test. It would be better to just test that a warning message occurred or perhaps the first few words of the text.

Reply:
The objective is to test that the warning is thrown.

Reply:
But it is asserting that the text matches exactly.